src/CMakeLists.txt

```diff
@@ -265,128 +268,126 @@ OMESSAGE("") # empty line
-	enable_testing()
+if (BUILD_TESTING)
+    # include(CTest) # calls enable_testing() and turns BUILD_TESTING to ON by default (among other useful things)
+    enable_testing()
 
-	add_subdirectory(tests)
-endif()
+    add_subdirectory(tests)
+endif ()
 
 OMESSAGE("")
 
 
-
-  
 # Informations for NSIS
-if(WIN32 OR WIN64)
-	if(MSVC)
-		if("${CMAKE_BUILD_TYPE}" STREQUAL "Release")
-			set(NSIS_TARGET "Release")
-		else()
-			set(NSIS_TARGET "Debug")
-		endif()
-		set(COMPILER_MARK "v")
-		#set(COMPILER_INCLUDE "vs9")
-	else(MSVC)
-		set(NSIS_TARGET "")
-		set(COMPILER_MARK "m")
-		set(COMPILER_INCLUDE "mingw")
-	endif(MSVC)
-	
-	set(CPACK_MODULE_PATH ${CMAKE_CURRENT_BINARY_DIR}/distrib/win32 ${CPACK_MODULE_PATH})
-
-	# Configure file with custom definitions for NSIS.
-	configure_file(
-		${PROJECT_SOURCE_DIR}/distrib/win32/version.cmake
-		${CMAKE_CURRENT_BINARY_DIR}/distrib/win32/version.nsh
-	)
-	
-	configure_file(
-		${PROJECT_SOURCE_DIR}/distrib/win32/build.template.cmake
-		${CMAKE_CURRENT_BINARY_DIR}/distrib/win32/build.nsh
-	)
-	
-	configure_file(
-		${PROJECT_SOURCE_DIR}/distrib/win32/make-zip-from-installer.cmake
-		${CMAKE_CURRENT_BINARY_DIR}/distrib/win32/make-zip-from-installer.bat
-	)
-	
-	SET(ANTARES_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR})
-
-	# Define system libraries so NSIS can package these files
-	include (InstallRequiredSystemLibraries)
-
-	#Convert to native and add double quote to be used in NSIS package
-	foreach(SYSTEM_RUNTIME_LIB ${CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS})
-		file(TO_NATIVE_PATH ${SYSTEM_RUNTIME_LIB} SYSTEM_RUNTIME_LIB_NATIVE)
-
-		list(APPEND SYSTEM_RUNTIME_LIBS_NATIVE \"${SYSTEM_RUNTIME_LIB_NATIVE}\")
-	endforeach()
-
-	#Replace ; by space so SYSTEM_RUNTIME_LIBS_STR can be used in File function in NSIS file
-	string (REPLACE ";" " " SYSTEM_RUNTIME_LIBS_STR "${SYSTEM_RUNTIME_LIBS_NATIVE}")
-
-	configure_file(
-		${PROJECT_SOURCE_DIR}/distrib/win32/NSIS.template.in
-		${CMAKE_CURRENT_BINARY_DIR}/distrib/win32/NSIS.template.in
-		@ONLY
-	)
-	
-	#Define cpack install script to checkout examples
-	configure_file(
-		${PROJECT_SOURCE_DIR}/distrib/CMakeLists.txt.in
-		${CMAKE_CURRENT_BINARY_DIR}/distrib/CMakeLists.txt
-		@ONLY
-	)
-
-	set(CPACK_INSTALL_SCRIPT ${CMAKE_CURRENT_BINARY_DIR}/distrib/CMakeLists.txt)
-
-	#Set package name from CMake and antares variables
-	set(CPACK_PACKAGE_FILE_NAME "rte-antares-${ANTARES_VERSION_HI}.${ANTARES_VERSION_LO}.${ANTARES_VERSION_REVISION}${ANTARES_VERSION_TAG}-installer-${ANTARES_INSTALLER_ARCH}")
-
-
-else()
-
-    if(CMAKE_SYSTEM_NAME MATCHES "Linux")
+if (WIN32 OR WIN64)
+    if (MSVC)
+        if ("${CMAKE_BUILD_TYPE}" STREQUAL "Release")
+            set(NSIS_TARGET "Release")
+        else ()
+            set(NSIS_TARGET "Debug")
+        endif ()
+        set(COMPILER_MARK "v")
+        #set(COMPILER_INCLUDE "vs9")
+    else (MSVC)
+        set(NSIS_TARGET "")
+        set(COMPILER_MARK "m")
+        set(COMPILER_INCLUDE "mingw")
```

Comment:
If we're going to format CMake files, we need a shared rule for it. Please revert changes related to formatting in this files.